apps/gitness/src/pages/pull-request/types/types.ts

```diff
@@ -152,3 +159,28 @@ export interface CommentItem<T = unknown> {
   content: string
   payload?: T // optional payload for callers to handle on callback calls
 }
+
+export interface ApprovalItem {
+  id: number
+  state?: string
+  method: string
+  title: string
+  items?: ApprovalItems[]
+}
+
+export interface ApprovalItems {
+  items: ApprovalItem[]
+}
+
+export type ButtonEnum = 'success' | 'muted' | 'default' | 'error' | 'warning' | null | undefined
+export interface FilterViewProps {
+  active?: string
+  currentUser: TypesUser
+  pullRequestMetadata?: TypesPullReq | undefined
+  reviewers?: ReviewerListPullReqOkResponse
+  submitReview?: (decision: PullReqReviewDecision) => void
+  refetchReviewers?: () => void
+  loading?: boolean
+}
+
+export type EnumPullReqReviewDecision = 'approved' | 'changereq' | 'pending' | 'reviewed' | 'outdated'
```

Comment:
Same as below

Reply:
Why not use existing type here? https://github.com/harness/react-api-client/blob/pkg-update-oct-10/packages/code-service/src/services/schemas/EnumPullReqReviewDecision.ts#L5

If any other value is needed, just extend or add to it (as a local declaration)